kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/model/instance/SolverInstance.java

```diff
@@ -38,11 +40,6 @@
     @XStreamAlias("score")
     private ScoreWrapper scoreWrapper;
 
-    @XmlElement(name = "planning-problem")
-    @XStreamAlias("planning-problem")
-    @XmlJavaTypeAdapter(JaxbUnknownAdapter.class)
-    private Object planningProblem;
-
```

Comment:
@mcimbora 
Thank you !!^^
I read only manual in drools web page, so I can't find this change.
New method work well^^